### DISCUSSION TITLE
not sure how to submit work

### DISCUSSION THREAD

**Discussion by Yuuuq** [Part-II-Computational-Physics/.github#2] [Q&A]
Dear Austen,

I hope this message finds you well. I'm reaching out to you because I think I may have made a mistake on our GitHub repository. I attempted to create a copy of the "Advection_equation_problem_set.ipynb" file by downloading it and then uploading it to the repository. Unfortunately, it seems that this action has replaced the old version with an updated one that is now broken and invalid.

I would appreciate it if you could offer me some guidance on how to rectify this situation, as I am still new to GitHub and may not be familiar with all of its functionalities. Thank you for your help in advance.

**Comment by AustenLamacraft** [comment]
If you're new to GitHub I'd suggest trying the [starter course](https://classroom.github.com/a/w9VOOLL8) first just to get a handle on what's going on. 

To undo the commits you've made you can try [this approach](https://stackoverflow.com/a/70245563/9032736) in the web interface
